codegen.ts

```diff
@@ -202,6 +195,11 @@ const { symbols } = Deno.dlopen({
   darwin: uri + "lib${name}.dylib",
   windows: uri + "${name}.dll",
   linux: uri + "lib${name}.so",
+  solaris: uri + "lib${name}.so",
```

Comment:
This seems unrelated to the PR. Could you revert this and split this off into another PR?

Reply:
Maybe there was a change to `/x/plug`. This seems to me to not be super future-proof. I think we should be using [name guessing](https://github.com/denosaurs/plug#example-using-automatic-binary-name-guessing) here.